dotnet-svcutil.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28728.38
+# Visual Studio Version 17
+VisualStudioVersion = 17.5.33530.505
```

Comment:
Is this change necessary for this PR?  What if someone is still on 16.0.x?

Reply:
This change is not necessary, I will revert.